.github/workflows/scheduled_test_cpu.yml

```diff
@@ -0,0 +1,52 @@
+name: (scheduled) Run tests on CPU
+
+on:
+  schedule:
```

Comment:
Maybe add `on: push` for these and remove from the pr workflow? @edgarriba

Reply:
To test had tests in all torch versions, instead just the same as in the PR

Reply:
Ok I see, sounds good